test/fake/fake-currency-plugin.js

```diff
@@ -271,6 +271,7 @@ class FakeCurrencyEngine {
 
   signTx(transaction: EdgeTransaction): Promise<EdgeTransaction> {
     transaction.txSecret = 'open sesame'
+    transaction.deviceDescription = 'iphone12'
```

Comment:
Delete this line. Instead, in currency-wallet-api.js / makeSpend, you should set up the deviceDescription there based on `input.props.state.login.deviceDescription`.